app/javascript/components/App.js

```diff
@@ -37,7 +40,42 @@ readSchool = () => {
   .catch(errors => console.log("School read errors:", errors))
 }
 
+createReview = (newReview) => {
+  fetch("http://localhost:3000/reviews", {
```

Comment:
Since this is a monolithic app, you will want to remove http://localhost:3000 from your fetch calls and just access "/reviews" and "/schools" to avoid running into cors errors.